Themis/Services/Submission/UnknownSubmissionServiceImpl.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UnknownSubmissionServiceImpl.swift
+//  Themis
+//
+//  Created by Tarlan Ismayilsoy on 23.05.23.
+//
+
+import Foundation
+import SharedModels
+import Common
+
+class UnknownSubmissionServiceImpl: SubmissionService {
+    
+    typealias SubmissionType = UnknownSubmission
+    
+    let defaultError = UserFacingError(title: "Exercise type not supported")
```

Comment:
The variable could be private

```suggestion
    private let defaultError = UserFacingError(title: "Exercise type not supported")
```